presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -2181,6 +2185,92 @@ public void testAggregationWithOrderBy()
                         Optional.empty()));
     }
 
+    @Test
+    public void testCreateRole()
+            throws Exception
+    {
+        assertStatement("CREATE ROLE role", new CreateRole(new Identifier("role"), Optional.empty(), Optional.empty()));
+        assertStatement("CREATE ROLE role1 WITH ADMIN admin",
```

Comment:
It does not seem according to standard that we may specify `WITH ADMIN <identifier>`

From `5WD-02-Foundation-2003-09.pdf` section 12.4
```
1) A <role definition> defines a role.
2) Let U be the current user identifier and R be the current role name.
3) Case:
a) If WITH ADMIN <grantor> is not specified, then
Case:
i) If U is not the null value, then let A be U.
ii) Otherwise, let A be R.
b) If WITH ADMIN CURRENT_USER is specified, then let A be U.
c) If WITH ADMIN CURRENT_ROLE is specified, then let A be R.
4) A role authorization descriptor is created that identifies that the role identified by <role name> has been
granted to A WITH ADMIN OPTION, with a grantor of “_SYSTEM”.
```

Reply:
The problem with this is that (for us at least) is that TD already supports this syntax in its Presto branded release, so the question is can we drop it now?

Reply:
What about this:
```
A role authorization descriptor is created that identifies that the role identified by <role name> has been
granted to A WITH ADMIN OPTION, with a grantor of “_SYSTEM”.
```

To me it is written in a way it may mean anything, so maybe this one allows this? What do you think? Can you please translate that to english?

Reply:
ignored for now